lib/nrf_inbuilt_key/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+zephyr_library()
+zephyr_library_sources_ifdef(CONFIG_NRF_INBUILT_KEY nrf_inbuilt_key.c)
```

Comment:
This ifdef should be removed as it is redundant with the ifdef in lib/CMakeLists.txt.

Reply:
Good point!